src/OpenProject.Revit/OpenProject.Revit.csproj

```diff
@@ -37,6 +37,9 @@
     <PackageReference Condition="'$(Revit2021)' == true" Include="Revit_All_Main_Versions_API_x64" Version="2021.0.0" />
     <PackageReference Condition="'$(Revit2020)' == true" Include="Revit_All_Main_Versions_API_x64" Version="2020.0.1" />
     <PackageReference Condition="'$(Revit2019)' == true" Include="Revit_All_Main_Versions_API_x64" Version="2019.0.1" />
+    <PackageReference Include="Serilog" Version="2.10.0" />
```

Comment:
Since the Serilog dependencies are added both to `Revit` and `Windows` projects, should we maybe add them (and the configuration) to the shared project?

Reply:
good point, will check it out.

Reply:
extracted the logging code to shared project and removed the dependencies in the other projects